[2022-09-18 18:42:05,389] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: bigquery_dag.write_query_big_query_table 2022-09-12T03:00:00+00:00 [queued]>
[2022-09-18 18:42:05,493] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: bigquery_dag.write_query_big_query_table 2022-09-12T03:00:00+00:00 [queued]>
[2022-09-18 18:42:05,495] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2022-09-18 18:42:05,497] {taskinstance.py:1018} INFO - Starting attempt 1 of 3
[2022-09-18 18:42:05,502] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2022-09-18 18:42:05,596] {taskinstance.py:1038} INFO - Executing <Task(BigQueryOperator): write_query_big_query_table> on 2022-09-12T03:00:00+00:00
[2022-09-18 18:42:05,642] {standard_task_runner.py:51} INFO - Started process 1979 to run task
[2022-09-18 18:42:05,699] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'bigquery_dag', 'write_query_big_query_table', '2022-09-12T03:00:00+00:00', '--job-id', '322', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_bigquery.py', '--cfg-path', '/tmp/tmp57ko5aau']
[2022-09-18 18:42:05,731] {standard_task_runner.py:76} INFO - Job 322: Subtask write_query_big_query_table
[2022-09-18 18:42:06,129] {logging_mixin.py:103} INFO - Running <TaskInstance: bigquery_dag.write_query_big_query_table 2022-09-12T03:00:00+00:00 [running]> on host 27377494ab8b
[2022-09-18 18:42:06,540] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Dave
AIRFLOW_CTX_DAG_ID=bigquery_dag
AIRFLOW_CTX_TASK_ID=write_query_big_query_table
AIRFLOW_CTX_EXECUTION_DATE=2022-09-12T03:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-12T03:00:00+00:00
[2022-09-18 18:42:06,563] {bigquery.py:680} INFO - Executing: 
        select * from githubarchive.day.20220914 where actor.id=66800817
            
[2022-09-18 18:42:06,830] {logging_mixin.py:103} WARNING - /home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py:2052 DeprecationWarning: This method is deprecated. Please use `BigQueryHook.insert_job` method.
[2022-09-18 18:42:07,012] {bigquery.py:1510} INFO - Inserting job airflow_1663526527005481_4e8e27051e4b7eda1ea1c14d0ba14e9a
[2022-09-18 18:42:08,331] {taskinstance.py:1396} ERROR - 403 Access Denied: Table githubarchive:day.20220914: User does not have permission to query table githubarchive:day.20220914, or perhaps it does not exist in location northamerica-northeast1.

(job ID: airflow_1663526527005481_4e8e27051e4b7eda1ea1c14d0ba14e9a)

                       -----Query Job SQL Follows-----                       

    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
   1:
   2:        select * from githubarchive.day.20220914 where actor.id=66800817
   3:            
    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1086, in _run_raw_task
    self._prepare_and_execute_task_with_callbacks(context, task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1260, in _prepare_and_execute_task_with_callbacks
    result = self._execute_task(context, task_copy)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1300, in _execute_task
    result = task_copy.execute(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/operators/bigquery.py", line 706, in execute
    encryption_configuration=self.encryption_configuration,
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 2186, in run_query
    job = self.insert_job(configuration=configuration, project_id=self.project_id)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/common/hooks/base_google.py", line 425, in inner_wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 1512, in insert_job
    job.result()
  File "/home/airflow/.local/lib/python3.6/site-packages/google/cloud/bigquery/job/query.py", line 1159, in result
    super(QueryJob, self).result(retry=retry, timeout=timeout)
  File "/home/airflow/.local/lib/python3.6/site-packages/google/cloud/bigquery/job/base.py", line 631, in result
    return super(_AsyncJob, self).result(timeout=timeout, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/google/api_core/future/polling.py", line 134, in result
    raise self._exception
google.api_core.exceptions.Forbidden: 403 Access Denied: Table githubarchive:day.20220914: User does not have permission to query table githubarchive:day.20220914, or perhaps it does not exist in location northamerica-northeast1.

(job ID: airflow_1663526527005481_4e8e27051e4b7eda1ea1c14d0ba14e9a)

                       -----Query Job SQL Follows-----                       

    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
   1:
   2:        select * from githubarchive.day.20220914 where actor.id=66800817
   3:            
    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
[2022-09-18 18:42:08,397] {taskinstance.py:1440} INFO - Marking task as UP_FOR_RETRY. dag_id=bigquery_dag, task_id=write_query_big_query_table, execution_date=20220912T030000, start_date=20220918T184205, end_date=20220918T184208
[2022-09-18 18:42:08,668] {local_task_job.py:118} INFO - Task exited with return code 1
[2022-09-18 19:40:13,309] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: bigquery_dag.write_query_big_query_table 2022-09-12T03:00:00+00:00 [queued]>
[2022-09-18 19:40:13,456] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: bigquery_dag.write_query_big_query_table 2022-09-12T03:00:00+00:00 [queued]>
[2022-09-18 19:40:13,459] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2022-09-18 19:40:13,462] {taskinstance.py:1018} INFO - Starting attempt 1 of 3
[2022-09-18 19:40:13,464] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2022-09-18 19:40:13,569] {taskinstance.py:1038} INFO - Executing <Task(BigQueryOperator): write_query_big_query_table> on 2022-09-12T03:00:00+00:00
[2022-09-18 19:40:13,598] {standard_task_runner.py:51} INFO - Started process 2859 to run task
[2022-09-18 19:40:13,655] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'bigquery_dag', 'write_query_big_query_table', '2022-09-12T03:00:00+00:00', '--job-id', '417', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_bigquery.py', '--cfg-path', '/tmp/tmpswwvzvx7']
[2022-09-18 19:40:13,692] {standard_task_runner.py:76} INFO - Job 417: Subtask write_query_big_query_table
[2022-09-18 19:40:14,067] {logging_mixin.py:103} INFO - Running <TaskInstance: bigquery_dag.write_query_big_query_table 2022-09-12T03:00:00+00:00 [running]> on host 27377494ab8b
[2022-09-18 19:40:14,489] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Dave
AIRFLOW_CTX_DAG_ID=bigquery_dag
AIRFLOW_CTX_TASK_ID=write_query_big_query_table
AIRFLOW_CTX_EXECUTION_DATE=2022-09-12T03:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-12T03:00:00+00:00
[2022-09-18 19:40:14,527] {bigquery.py:680} INFO - Executing: 
        select * from githubarchive.day.20220914 where actor.id=66800817
            
[2022-09-18 19:40:14,658] {logging_mixin.py:103} WARNING - /home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py:2052 DeprecationWarning: This method is deprecated. Please use `BigQueryHook.insert_job` method.
[2022-09-18 19:40:14,828] {bigquery.py:1510} INFO - Inserting job airflow_1663530014811629_4e8e27051e4b7eda1ea1c14d0ba14e9a
[2022-09-18 19:40:16,502] {taskinstance.py:1396} ERROR - 403 Access Denied: Table githubarchive:day.20220914: User does not have permission to query table githubarchive:day.20220914, or perhaps it does not exist in location northamerica-northeast1.

(job ID: airflow_1663530014811629_4e8e27051e4b7eda1ea1c14d0ba14e9a)

                       -----Query Job SQL Follows-----                       

    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
   1:
   2:        select * from githubarchive.day.20220914 where actor.id=66800817
   3:            
    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1086, in _run_raw_task
    self._prepare_and_execute_task_with_callbacks(context, task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1260, in _prepare_and_execute_task_with_callbacks
    result = self._execute_task(context, task_copy)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1300, in _execute_task
    result = task_copy.execute(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/operators/bigquery.py", line 706, in execute
    encryption_configuration=self.encryption_configuration,
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 2186, in run_query
    job = self.insert_job(configuration=configuration, project_id=self.project_id)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/common/hooks/base_google.py", line 425, in inner_wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 1512, in insert_job
    job.result()
  File "/home/airflow/.local/lib/python3.6/site-packages/google/cloud/bigquery/job/query.py", line 1159, in result
    super(QueryJob, self).result(retry=retry, timeout=timeout)
  File "/home/airflow/.local/lib/python3.6/site-packages/google/cloud/bigquery/job/base.py", line 631, in result
    return super(_AsyncJob, self).result(timeout=timeout, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/google/api_core/future/polling.py", line 134, in result
    raise self._exception
google.api_core.exceptions.Forbidden: 403 Access Denied: Table githubarchive:day.20220914: User does not have permission to query table githubarchive:day.20220914, or perhaps it does not exist in location northamerica-northeast1.

(job ID: airflow_1663530014811629_4e8e27051e4b7eda1ea1c14d0ba14e9a)

                       -----Query Job SQL Follows-----                       

    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
   1:
   2:        select * from githubarchive.day.20220914 where actor.id=66800817
   3:            
    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
[2022-09-18 19:40:16,555] {taskinstance.py:1440} INFO - Marking task as UP_FOR_RETRY. dag_id=bigquery_dag, task_id=write_query_big_query_table, execution_date=20220912T030000, start_date=20220918T194013, end_date=20220918T194016
[2022-09-18 19:40:16,792] {local_task_job.py:118} INFO - Task exited with return code 1
[2022-09-18 20:17:48,856] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: bigquery_dag.write_query_big_query_table 2022-09-12T03:00:00+00:00 [queued]>
[2022-09-18 20:17:48,975] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: bigquery_dag.write_query_big_query_table 2022-09-12T03:00:00+00:00 [queued]>
[2022-09-18 20:17:48,979] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2022-09-18 20:17:48,981] {taskinstance.py:1018} INFO - Starting attempt 1 of 3
[2022-09-18 20:17:48,983] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2022-09-18 20:17:49,122] {taskinstance.py:1038} INFO - Executing <Task(BigQueryOperator): write_query_big_query_table> on 2022-09-12T03:00:00+00:00
[2022-09-18 20:17:49,163] {standard_task_runner.py:51} INFO - Started process 1956 to run task
[2022-09-18 20:17:49,268] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'bigquery_dag', 'write_query_big_query_table', '2022-09-12T03:00:00+00:00', '--job-id', '481', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_bigquery.py', '--cfg-path', '/tmp/tmp5cvhffu2']
[2022-09-18 20:17:49,318] {standard_task_runner.py:76} INFO - Job 481: Subtask write_query_big_query_table
[2022-09-18 20:17:49,708] {logging_mixin.py:103} INFO - Running <TaskInstance: bigquery_dag.write_query_big_query_table 2022-09-12T03:00:00+00:00 [running]> on host 27377494ab8b
[2022-09-18 20:17:50,179] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Dave
AIRFLOW_CTX_DAG_ID=bigquery_dag
AIRFLOW_CTX_TASK_ID=write_query_big_query_table
AIRFLOW_CTX_EXECUTION_DATE=2022-09-12T03:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-12T03:00:00+00:00
[2022-09-18 20:17:50,264] {bigquery.py:680} INFO - Executing: 
        select * from githubarchive.day.20220914 where actor.id=66800817
            
[2022-09-18 20:17:50,598] {logging_mixin.py:103} WARNING - /home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py:2052 DeprecationWarning: This method is deprecated. Please use `BigQueryHook.insert_job` method.
[2022-09-18 20:17:50,957] {bigquery.py:1510} INFO - Inserting job airflow_1663532270933249_4e8e27051e4b7eda1ea1c14d0ba14e9a
[2022-09-18 20:17:52,495] {taskinstance.py:1396} ERROR - 403 Access Denied: Table githubarchive:day.20220914: User does not have permission to query table githubarchive:day.20220914, or perhaps it does not exist in location northamerica-northeast1.

(job ID: airflow_1663532270933249_4e8e27051e4b7eda1ea1c14d0ba14e9a)

                       -----Query Job SQL Follows-----                       

    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
   1:
   2:        select * from githubarchive.day.20220914 where actor.id=66800817
   3:            
    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1086, in _run_raw_task
    self._prepare_and_execute_task_with_callbacks(context, task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1260, in _prepare_and_execute_task_with_callbacks
    result = self._execute_task(context, task_copy)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1300, in _execute_task
    result = task_copy.execute(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/operators/bigquery.py", line 706, in execute
    encryption_configuration=self.encryption_configuration,
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 2186, in run_query
    job = self.insert_job(configuration=configuration, project_id=self.project_id)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/common/hooks/base_google.py", line 425, in inner_wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 1512, in insert_job
    job.result()
  File "/home/airflow/.local/lib/python3.6/site-packages/google/cloud/bigquery/job/query.py", line 1159, in result
    super(QueryJob, self).result(retry=retry, timeout=timeout)
  File "/home/airflow/.local/lib/python3.6/site-packages/google/cloud/bigquery/job/base.py", line 631, in result
    return super(_AsyncJob, self).result(timeout=timeout, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/google/api_core/future/polling.py", line 134, in result
    raise self._exception
google.api_core.exceptions.Forbidden: 403 Access Denied: Table githubarchive:day.20220914: User does not have permission to query table githubarchive:day.20220914, or perhaps it does not exist in location northamerica-northeast1.

(job ID: airflow_1663532270933249_4e8e27051e4b7eda1ea1c14d0ba14e9a)

                       -----Query Job SQL Follows-----                       

    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
   1:
   2:        select * from githubarchive.day.20220914 where actor.id=66800817
   3:            
    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
[2022-09-18 20:17:52,562] {taskinstance.py:1440} INFO - Marking task as UP_FOR_RETRY. dag_id=bigquery_dag, task_id=write_query_big_query_table, execution_date=20220912T030000, start_date=20220918T201748, end_date=20220918T201752
[2022-09-18 20:17:52,705] {local_task_job.py:118} INFO - Task exited with return code 1
